convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -75,5 +97,16 @@ def run(self):
                 "Consider updating the packages before proceeding with the conversion."
                 % (len(packages_to_update), repos_message, " ".join(packages_to_update))
             )
+            self.add_message(
+                level="WARNING",
+                id="OUT_OF_DATE_PACKAGES",
+                message=(
+                    "The system has %s package(s) not updated based %s.\n"
+                    "List of packages to update: %s.\n\n"
+                    "Not updating the packages may cause the conversion to fail.\n"
+                    "Consider updating the packages before proceeding with the conversion."
+                    % (len(packages_to_update), repos_message, " ".join(packages_to_update))
+                ),
+            )
```

Comment:
Remember to remove the logger.warning above

```suggestion
            message = (
                    "The system has %s package(s) not updated based %s.\n"
                    "List of packages to update: %s.\n\n"
                    "Not updating the packages may cause the conversion to fail.\n"
                    "Consider updating the packages before proceeding with the conversion."
                    % (len(packages_to_update), repos_message, " ".join(packages_to_update))
                )
            logger.warning(message)
            self.add_message(
                level="WARNING",
                id="OUT_OF_DATE_PACKAGES",
                message=message,
            )
```